include/nnvm/op_attr_types.h

```diff
@@ -151,6 +151,21 @@ using FInplaceIdentity = std::function<std::vector<bool> (const NodeAttrs& attrs
 using FIgnoreInputs = std::function<
   std::vector<uint32_t> (const NodeAttrs& attrs)>;
 
+/*!
+ * \brief Get the output node of the op node
+ *  This function generates the graph of the node
+ * \param nodeptr The current node
+ * \param inputs The current node's inputs
+ * \param input_shapes The current node's input shapes
+ * \return the Node's outputs
+ *
+ * \note Register under "FCompute"
+ */
```

Comment:
Rename to FExpandCompute, because fcompute was used by downstream for getting compute kernels